main.c

```diff
@@ -67,6 +28,11 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void testPrintManual(void)
```

Comment:
testPrintManual is not called anywhere? How do you know if `PrintColourCode` works?